src/OpenTelemetry.AutoInstrumentation/Logging/OtelLogging.cs

```diff
@@ -89,37 +123,53 @@ internal static long GetConfiguredFileSizeLimitBytes()
 
     private static IOtelLogger CreateLogger(string suffix)
     {
-        ISink? sink = null;
-
-        if (!ConfiguredLogLevel.HasValue)
+        if (!_configuredLogLevel.HasValue)
         {
             return NoopLogger.Instance;
         }
 
-        try
+        var sink = CreateSink(suffix);
+
+        return new InternalLogger(sink, _configuredLogLevel.Value);
+    }
+
+    private static ISink CreateSink(string suffix)
+    {
+        if (_configuredLogSink == LogSink.NoOp)
         {
-            var logDirectory = GetLogDirectory();
-            if (logDirectory != null)
+            return new NoopSink();
+        }
+        else if (_configuredLogSink == LogSink.File)
+        {
+            try
+            {
+                var logDirectory = GetLogDirectory();
+                if (logDirectory != null)
+                {
+                    var fileName = GetLogFileName(suffix);
+                    var logPath = Path.Combine(logDirectory, fileName);
+
+                    return new RollingFileSink(
+                        path: logPath,
+                        fileSizeLimitBytes: FileSizeLimitBytes,
+                        retainedFileCountLimit: 10,
+                        rollingInterval: RollingInterval.Day,
+                        rollOnFileSizeLimit: true,
+                        retainedFileTimeLimit: null);
+                }
+            }
+            catch (Exception)
             {
-                var fileName = GetLogFileName(suffix);
-                var logPath = Path.Combine(logDirectory, fileName);
-                sink = new RollingFileSink(
-                    path: logPath,
-                    fileSizeLimitBytes: FileSizeLimitBytes,
-                    retainedFileCountLimit: 10,
-                    rollingInterval: RollingInterval.Day,
-                    rollOnFileSizeLimit: true,
-                    retainedFileTimeLimit: null);
+                // unable to configure logging to a file
             }
         }
-        catch (Exception)
+        else if (_configuredLogSink == LogSink.Console)
         {
-            // unable to configure logging to a file
+            return new ConsoleSink(suffix);
         }
 
-        sink ??= new NoopSink();
-
-        return new InternalLogger(sink, ConfiguredLogLevel.Value);
+        // Default to NoopSink
+        return new NoopSink();
```

Comment:
Is this consistent with the native default?

Reply:
Wondering this too: reading https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3558/files#diff-6576a72e1be921af76d4e4e9525cb42dff458c51b1a1439807de36371a31fa1aR296 makes me think `file` is the default fallback in the profiler.

Reply:
File is the default sink, the default fallback sink is NoOp. This might cause the confusion at first.

Reply:
I'm open to considering changing the default, but, the question is to ensure that we are consistent between defaults and fallbacks for the logs on both native and managed. What is the current code and the change doing in this regard?

Reply:
I updated the "What" section about what is the current logic, defaults, fallbacks and what's added.